db/schema.ts

```diff
@@ -0,0 +1,24 @@
+import { Message } from "ai";
+import { InferSelectModel } from "drizzle-orm";
+import { pgTable, varchar, timestamp, json, uuid } from "drizzle-orm/pg-core";
+
+export const user = pgTable("User", {
+  id: uuid("id").primaryKey().notNull().defaultRandom(),
+  email: varchar("email", { length: 64 }).notNull(),
+  password: varchar("password", { length: 64 }),
+});
+
+export type User = InferSelectModel<typeof user>;
+
+export const chat = pgTable("Chat", {
+  id: uuid("id").primaryKey().notNull().defaultRandom(),
```

Comment:
what do you think about using CUID instead of UUID? it gives adequate protection but with much much more readable / application friendly ids. 

Here's the example file for generating id:
https://github.com/vercel-labs/ai-sdk-preview-rag/blob/b9399d53f8a204cce6a5fe76d98bb0a329851c5f/lib/utils.ts#L9

and then how it's used in schemas:
https://github.com/vercel-labs/ai-sdk-preview-rag/blob/b9399d53f8a204cce6a5fe76d98bb0a329851c5f/lib/db/schema/embeddings.ts#L10

Reply:
some more info 
https://www.unkey.com/blog/uuid-ux

Reply:
agree with the ux benefits, but I feel the template can default to `uuid` since it's a more familiar id format (chatgpt and claude uses them) and allow developers to choose their own id format if they choose to customize